tools/lib/provision.py

```diff
@@ -247,12 +247,26 @@ def main(options):
         run(["sudo", "service", "redis-server", "restart"])
         run(["sudo", "service", "memcached", "restart"])
     if not options.is_production_travis:
-        # These won't be used anyway
-        run(["scripts/setup/configure-rabbitmq"])
-        # Need to set up Django before using is_template_database_current.
+
+        # Need to set up Django before using is_template_database_current and rabbitmq credentials
         os.environ.setdefault("DJANGO_SETTINGS_MODULE", "zproject.settings")
         import django
         django.setup()
+
+        import pika
+        from django.conf import settings
+        # These won't be used anyway
+        # check if rabbitmq is already configured
+        try:
+            credentials = pika.PlainCredentials(settings.RABBITMQ_USERNAME,
+                                                settings.RABBITMQ_PASSWORD)
+            pika.BlockingConnection(pika.ConnectionParameters(settings.RABBITMQ_HOST,
```

Comment:
Don't we already import Django here? Just after the rabbitmq setup, in the existing version of this script.

Or maybe it's subtler just what we have to avoid importing.